tests/test_netcdf_monitor.py

```diff
@@ -25,6 +26,67 @@
 }
 
 
+class NetCDFMonitorAliasTests(unittest.TestCase):
+
+    def setUp(self):
+        self.ncfile = 'out.nc'
+
+    def tearDown(self):
+        if os.path.isfile(self.ncfile):
+            os.remove(self.ncfile)
+
+    def store_state_and_check_file(self, aliases):
+        assert not os.path.isfile(self.ncfile)
+        monitor = NetCDFMonitor(self.ncfile, aliases=aliases,
+                                write_on_store=True)
+        monitor.store(state)
```

Comment:
Where is `state` defined? It should probably be passed in as an argument, or stored as `self.state` in `setUp` and then accessed using `self.state`.

Reply:
`state` is defined (by you) outside the class, at the top of the module. That state is used by other tests throughout the module so I figured I would use it for mine as well.

Reply:
OK, I'd say this is fine. It just was very unclear to me because github hid that part of the code.